builder/builder.go

```diff
@@ -46,6 +46,8 @@ type BuildOptions struct {
 	ID string
 	// Repository is the repo to build.
 	Repository string
+	// github.com or enterprise url
+	GithubDomain string
```

Comment:
This doesn't belong in this struct. This should be supplied to the [docker.Builder](https://github.com/remind101/conveyor/blob/master/builder/docker/docker.go#L43-L57) during initialization.

I think adding an `Env map[string]string` field to that struct would be nice and then we can set the github domain like this:

```go
builder := docker.NewBuilder()
builder.Env["GITHUB_DOMAIN"] = c.String("github.domain")
```